sharding/partitions.go

```diff
@@ -233,19 +237,22 @@ func (p *Partitions) updateRemote(nodes []string) {
 }
 
 func (p *Partitions) updateMissing() {
-	// Check for each partition. If every one is available on at least one node,
+	// Check for each partition. If every one is available on at least minReplication node,
 	// then we're ready to rumble.
 	missing := 0
```

Comment:
Maybe rename this to `underReplicated` (there's a few occurrences/comments referencing `missing`/`Missing` in this file).